geoapps/utils/utils.py

```diff
@@ -41,6 +41,38 @@
 from sklearn.neighbors import KernelDensity
 
 
+def sorted_children_dict(object: UUID | Entity):
+
+    if isinstance(object, UUID):
+        object = workspace.get_entity(object)[0]
+
+    children_dict = {}
+    iters = []
+    iteration_data = []
+    other_data = []
+    for c in object.children:
+        if not isinstance(c, (IntegerData, FloatData)):
+            continue
+        else:
+            children_dict[c.name] = c.uid
+            name = c.name
+            syllables = name.split("_")
+            if any([n.lower() == "iteration" for n in syllables]):
```

Comment:
This is a bit too target to inversion. Consider using a regexp to detect any float or integer. Something like 
`re.findall(""[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?", name)` should return all decimal numbers including exponential format from the string `name`.

Reply:
fair enough, done.